home-manager/home-manager

```diff
@@ -29,7 +29,10 @@ function removeByName() {
 }
 
 function setNixProfileCommands() {
-    if [[ -e $HOME/.nix-profile/manifest.json ]] ; then
+    if  [[ -e $HOME/.nix-profile/manifest.json 
+        || -e $XDG_STATE_HOME/nix/profile/manifest.json 
+        || -e $HOME/.local/state/nix/profile/manifest.json ]] ; then
```

Comment:
This is more correct according to [the spec](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html):

```suggestion
        || -e ${XDG_STATE_HOME:-$HOME/.local/state}/nix/profile/manifest.json ]] ; then
```